### DISCUSSION TITLE
Additional Template for Type in Text

### DISCUSSION THREAD

**Discussion by LurkingStocks** [ikkz/anki-template#50] [Ideas]
@ikkz
In Anki, there is already a template that allows you to enter text into a field and then, during review, shows how well the answer matches the text in the {{Back}} field. However, I don’t think this feature is very well implemented. The text input field is not very clear, the font of the displayed correction cannot be changed, and it has a different style.

Additionally, your features with ChatGPT and Google search are very helpful and offer a lot of possibilities.

I imagine a template roughly like this:

![image](https://github.com/user-attachments/assets/626f29f3-e3b3-46c9-aa1f-bf72d0093f62)

It would resemble your current setup.
It would be ideal to have a Question Column, followed by the entered text below it.
In a third column, the verification of the entered text could be displayed, with incorrect parts marked in red and correct ones in green.
Everything else could remain as it is in the templates you’ve created so far.

What do you think about this?

**Comment by ikkz** [comment]
I believe it can be implemented. 

The GitHub Discussions module is currently not in use. You can click "Create issue from discussion" on the right side of this discussion to move it to the Issues module.
